README.md

```diff
@@ -5,42 +5,129 @@
  <p align="center">Helper library to sign a transaction
 </p>
 
-## Install
+The prefix `\x19LSP6 ExecuteRelayCall:\n` is added to prevent users from signing a transaction while thinking it's a message.
```

Comment:
We could do with more context here. Why do we use a prefix here? The prefix is added to what? 

You could have a sentence saying this library is used to sign transactions so they are ready to be passed to the LSP6 Execute Relay Call function. This library will add the LSP6 ExecuteRelayCall prefix to a message and sign it